src/Tizen.NUI/src/public/Utility/Palette.cs

```diff
@@ -48,7 +48,7 @@ namespace Tizen.NUI
     ///     // Use generated instance
```

Comment:
the line above is no longer correct, the GenerateAsync method has been changed

also - most of this description should go to \<remarks/\> section - summary should be short, so it would be \<summary\>....\</summary\>\<remarks\>...\</remarks\>
the code examples will not be formatted as in this code, please add \<br/\>in lines 43, 46, 47, 48, 49

Reply:
Thanks for the guide. The contents will be updated separately on Doc git.